app/views/unattended/report_templates/ansible_-_ansible_inventory.erb

```diff
@@ -111,14 +111,16 @@ model: ReportTemplate
 require:
 - plugin: foreman_ansible
   version: '3.1.0'
-- plugin: katello
-  version: '3.9.0'
 -%>
 <%- report_headers 'id', 'name' -%>
-<%- content_attrs = input('Content Attributes') == 'yes' %>
+<%- katello_available = plugin_present?('katello') %>
+<%- content_attrs = katello_available && input('Content Attributes') == 'yes' %>
```

Comment:
would it make sense to:
```
<%- if katello_available -%>
<%-   content_attrs = katello_available && input('Content Attributes') == 'yes' -%>
<%-   include_host_collections = katello_available && input('Host Collections') == 'yes' -%>
<%- end -%>
```

that way it's visually clear, what params are only applicable with katello

Reply:
Aren't `content_attrs` and `include_host_collections` then undefined when Katello is not available? Or it that OK in erb and undefined vars are automatically `nil`?

Reply:
Seems to work, okay…